app/assets/javascripts/fe/fe.public.nojquery.js.erb

```diff
@@ -2,6 +2,416 @@
 
 // used by answer sheets
 
+window.fe = {};
```

Comment:
Btw @jbirdjavi @knutsenm the reason I moved this to top-level js is because with Justin's change

```
-$(function() {
+$(document).on('ready turbo:load', function () {
````

in https://github.com/CruGlobal/fe/pull/143/files, it changed the order that things were loading, and code that expected window.fe to be set was crashing as it wasn't set by then.

Reply:
Yeah I caught that. We'll want to test Global 360 with this change, but I think it'll be fine.